tools/runut.cmd

```diff
@@ -5,6 +5,7 @@ rem Run the console unit tests.
 call %TAEF% ^
     %OPENCON%\bin\%PLATFORM%\%_LAST_BUILD_CONF%\Conhost.Unit.Tests.dll ^
     %OPENCON%\bin\%PLATFORM%\%_LAST_BUILD_CONF%\TextBuffer.Unittests.dll ^

```

Comment:
Oh no. I know this isn't a part of this PR, but if the TextBuffer unit tests aren't named `.Unit.Tests.` then I'm pretty sure the CI isn't running them.

Reply:
I wrote this so I could turn it into an issue. I've done that.